src/main/java/de/dentrassi/rpm/builder/RpmMojo.java

```diff
@@ -691,6 +688,21 @@ public void setSignatureConfiguration ( final String signatureConfiguration )
         this.signatureConfiguration = signatureConfiguration;
     }
 
+    /**
+     * The prefix to use when reading signing variables
+     * from Maven's settings.xml.
+     *
+     * The signing variables are: keyId, keyringFile, passphrase, hashAlgorithm.
+     */
+    @Parameter(defaultValue = "rpm.")
+    private String signCfgPrefix;
```

Comment:
I would prefer a non-abbreviated name, e.g. `signConfigurationPrefix`. And wouldn't it make sense to allow the user to set this via properties as well?

Reply:
I can certainly do that. I just took the existing syntax from jdeb, but I can update it...